src/main/java/com/craftaro/epicfarming/farming/FarmManager.java

```diff
@@ -28,6 +31,11 @@ public void addFarm(Location location, Farm farm) {
 
     public void addFarms(Collection<Farm> farms) {
         for (Farm farm : farms) {
+            //System.out.println("Registering Farm ID: " + farm.getId() + " at " + farm.getLocation()); // Debugging
```

Comment:
> Personally not a big fan of commented-out code, especially if they mostly look temporary.

Reply:
all  temporary debug has been removed instead of commenting it out